build.sh

```diff
@@ -144,33 +137,36 @@ if (( ${NUMARGS} == 0 )) || hasArg libcuml || hasArg prims || hasArg bench; then
 fi
 
 # Run all make targets at once
+
 MAKE_TARGETS=
-if (( ${NUMARGS} == 0 )) || hasArg libcuml; then
```

Comment:
I just want to confirm my understanding: `MAKE_TARGETS=""` will result in the same behavior as running `build.sh` with no args before (ie. build basically everything by default)? I just need to know since that affects the RAPIDS Dockerfiles and `BUILD.md` if not.

Reply:
yes, ./build.sh without arguments will build everything essentially by default, it just works better for the `make` commands to print a little bit less repeated configuration stuff from the subprojects